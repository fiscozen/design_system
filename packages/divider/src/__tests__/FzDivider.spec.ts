import { describe, it, vi } from "vitest";
import { mount } from "@vue/test-utils";
import { FzDivider } from "..";

describe.concurrent("FzDivider", () => {
  it("matches snaphost", async ({ expect }) => {
    const wrapper = mount(FzDivider, {
      props: {},
      slots: {},
    });

    expect(wrapper.html()).toMatchSnapshot();
  });
});
