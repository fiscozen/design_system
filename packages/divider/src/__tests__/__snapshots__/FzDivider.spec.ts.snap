// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FzDivider > matches snaphost 1`] = `
"<div class="w-full my-16 relative">
  <hr>
  <!--v-if-->
</div>"
`;
